chat-plugins/github.js

```diff
@@ -0,0 +1,94 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
+*/
+'use strict';
+
+let config = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
+	config.port = 3420;
+	config.secret = "";
+} else {
+	config = {
+		port: Config.github.port,
+		secret: Config.github.secret,
+	};
+}
+
+let git = exports.github = require('githubhook')(config);
+
+let updates = {};
+
+let sendReport = function (html) {
+	if (Config.github && Config.github.rooms) {
+		Object.keys(Config.github.rooms).forEach(room => {
+			let r = Rooms(room);
+			if (!r) return;
+			r.add(`|html|<div class="infobox">${html}</div>`);
+		});
+	} else if (Rooms('development')) {
+		Rooms('development').add(`|html|<div class="infobox">${html}</div>`);
+	}
+};
+
+git.on('push', (repo, ref, result) => {
```

Comment:
This shouldn't be an arrow function.

``Arrow functions: ALWAYS - Supported in Node 4+, good performance. Obviously use only for callbacks; don't use in situations where this shouldn't be bound.``